pegasus/sites.v3/code.org/public/maker/circuitplayground.haml

```diff
@@ -0,0 +1,97 @@
+          .text-wrapper
+            %h3 Works with App Lab
+            %p.no-margin-bottom Using App Lab students can quickly build apps that communicate with external hardware using our block-to-text app development environment.
+      %figure.col-25{style: "margin-top: 2em; text-align: center"}
+        %img{src: "/images/maker/circuitplayground-no-bg.gif", alt: ""}
+    .clear
+
+%section.bg-neutral-light.device-setup
+  .wrapper
+    %h2 Device setup requirements
+    %p The Circuit Playground is a series of circuit boards produced by Adafruit with a variety of sensors and outputs built in.
+    %p <strong>There are three variants of the Circuit Playground, two are compatible with App Lab and one of them is not:</strong>
+    %ul.icon-bullet-list{style: "margin-top: 0"}
+      %li
+        %div
+          %i{class: "#{icon_circle_check}", style: "color: #3EA33E"}
+        .text-wrapper
+          %h3 The Circuit Playground Classic
+      %li
+        %div
+          %i{class: "#{icon_circle_check}", style: "color: #3EA33E"}
+        .text-wrapper.no-margin-bottom
+          %h3 The Circuit Playground Express
+      %li
+        %div
+          %i{class: "#{icon_circle_xmark}", style: "color: #E5311A"}
+        .text-wrapper.no-margin-bottom
+          %h3 The Circuit Playground Bluefruit
+    %p Wondering which version you have? <a href="https://learn.adafruit.com/introducing-circuit-playground/classic-vs-express" target="_blank" rel="noopener noreferrer">Check out Adafruit's guide</a>.
+
+%section.bg-primary-light
+  .wrapper
+    %p.no-margin-bottom
+      %i{class: "#{icon_flag}", style: "margin-right: 4px"}
+      %strong
+        Once you have your device, <a href="https://studio.code.org/maker/setup">visit the Maker Setup page</a> to connect your Circuit Playground with App Lab.
+
+%section
+  .wrapper
+    %h2 Getting the Circuit Playground for your classroom
+    .action-block__wrapper.action-block__wrapper--two-col
+      .action-block.action-block--one-col
+        %h3 Circuit Playground Classroom Kit
+        %img{src: "/images/maker/maker-cp-kit.png", alt: "", style: "width: 100%"}
+        %p Contains Circuit Playground boards and accessories necessary for using Code.org's curriculum with a class or 30 students.
+        %p{style: "height: 83px"} All educators receive a 10% discount when they apply the code <strong>ADAEDU</strong> at checkout. <em>Note:</em> The ADAEDU code only applies to orders of $250 or greater.
+        %a.link-button{href: "https://www.adafruit.com/product/3399", target: "_blank", rel: "noopener noreferrer"} Get the classroom kit
+      .action-block.action-block--one-col
+        %h3 Circuit Playground Individual Kit
+        %img{src: "/images/maker/maker-cp-individual.png", alt: "", style: "width: 100%"}
+        %p{style: "height: 160px"} For smaller classrooms, we recommend investing in a set of Circuit Playground Individual Kits. The individual kit is designed for a single or pair of students.
```

Comment:
I know I'm not supposed to be copy-editing but this parses weird because you don't say "a single students" ... should it be "a single student or pair of students"

Reply:
I appreciate your copy-edits 😄 I'll update it to "a single student or pair of students".